index.html

```diff
@@ -0,0 +1,208 @@
+                <div class="plane">
+                  <img src="images/square.svg" alt="square">
+                </div>
+              </div>
+               <div class="section6">
+                <article class="quote">
+                  <div class="holder-text">
+                    <img src="images/holder.svg" alt="holder">
+                    <div class="holder-text1">
+                      <img src="images/ellipse.svg" alt="holder">
+                      <ul>
+                        <li class="Language">
+                          <p>Languages</p>
+                          </li>
+                            <ul class="media">
+                              <li>JavaScript</li>
+                              <li>Ruby</li>
+                              <li>HTML</li>
+                              <li>CSS</li>
+                            </ul>
+                      </ul>              
+                    </div>
+                  </div>
+                  </article>
+                    <section class="quote">
+                        <div class="holder-text">
+                        <img src="images/holder.svg" alt="holder">
+                        <div class="holder-text1">
+                          <img src="images/ellipse2.svg" alt="holder">
+                          <ul>
+                            <li class="Language">
+                              <p>Frameworks</p>
+                            </li>
+                            <ul class="media">
+                              <li>React.js</li>
+                              <li>Ruby on Rails</li>
+                              <li>RSpec</li>
+                              <li>Capybara</li>
+                              <li>Selenium</li>
+                            </ul>
+                          </ul>                   
+                          </div>
+                       </div>
+                        </section>
+                        <article class="quote">
+                            <div class="holder-text">
+                            <img src="images/holder.svg" alt="holder">
+                            <div class="holder-text1">
+                              <img src="images/skills.svg" alt="holder">
+                              <ul>
+                                <li class="Language">
+                                  <p>Skills</p>
+                                </li>
+                                <ul class="media">                                                  
+                                  <li>Database Management</li>
+                                  <li>Version Controle</li>
+                                  <li>CLI</li>
+                                  <li>WEb Development</li>
+                                  <li>API Design</li>
+                                <ul>
+                              </ul>
+                             
+                              </div>
+                            </div>
```

Comment:
- [ ] For better HTML semantics I'd suggest you place the technology categories ("Languages", "Frameworks" and "Skills") in a `ul` tag, and for each of them in a `li` tag. This means you should replace the `div` tag at line `144` with a `ul` tag and for each `section` & `article` tags you used should be replaced with an `li` tag.